src/main/java/pom/NaptolCartPage.java

```diff
@@ -0,0 +1,123 @@
+package pom;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.poi.EncryptedDocumentException;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import utility.Parameterization;
+
+public class NaptolCartPage extends BaseClass{
+
+	
+	@FindBy(xpath = "//ul[@id='cartData']//h2")private WebElement CartProductName;
```

Comment:
All element name should be in camel case

Reply:
Fixed